tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/core/portal/PortalBundle.java

```diff
@@ -30,15 +31,17 @@
 
     IPath getLiferayHome();
 
-    String getMainClass();
+    String getStartMainClass();
+    
+    String getStopMainClass();
 
     IPath getModulesPath();
 
-    IPath[] getRuntimeClasspath();
+    IPath[] getRuntimeClasspath(IVMInstall vmInstall);
```

Comment:
Ok we can leave that portion in then.

On Fri, May 8, 2015 at 10:28 AM, simon.jiang <notifications@github.com>
wrote:

> In
> tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/core/portal/PortalBundle.java
> <https://github.com/gamerson/liferay-ide/pull/989#discussion_r29913411>:
>
> >
> >      IPath getModulesPath();
> >
> > -    IPath[] getRuntimeClasspath();
> > +    IPath[] getRuntimeClasspath(IVMInstall vmInstall);
>
> Because glassfish and jetty all need jdk lib to help them start and stop.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gamerson/liferay-ide/pull/989/files#r29913411>.
>



-- 
Greg Amerson
Liferay Developer Tools
Liferay, Inc. www.liferay.com